src/app/cli/src/coda.ml

```diff
@@ -46,6 +46,21 @@ let chain_id ~genesis_state_hash ~genesis_constants =
 [%%inject
 "compile_time_current_protocol_version", current_protocol_version]
 
+[%%if
```

Comment:
I think you can use `ifdef plugins`, then you only have to define it in the profiles where it's needed.

Reply:
I get an error from optcomp, where it asks for `[%%undef plugins]` in the config :/